### ISSUE TITLE
PvP Rewards - manual values as reward

### ISSUE THREAD

**Issue by Helias** [azerothcore/acore-cms#46]
It would be really nice to allow the user to put a specific value per each players via input text bot next to it

![image](https://user-images.githubusercontent.com/519778/120693625-841de180-c4a9-11eb-9787-81dc130fa0d6.png)

if we can edit the Extra Points manually would be perfect, because it's not possible to use a fixed formula for the rewards.

**Comment by nicoaravena** [comment]
I was reading about this and you can give a custom amount with the mycred addon while editing a profile as administrator. So there is no need to add more complexity to this view in this case.

**Comment by Helias** [comment]
Yes, indeed I already do this in that way, but it's not so comfortable, but if this takes too much effort to do, we can even lave like this and give priority to others features
